package.json

```diff
@@ -26,6 +26,7 @@
     "@angular/router": "16.1.7",
     "@ngneat/transloco": "^5.0.2",
     "@nguniversal/express-engine": "^16.1.1",
+    "@ngx-translate/core": "^15.0.0",
```

Comment:
No need for this. I've already setup the translations just sync the latest changes